blockchain/blockchain_test.go

```diff
@@ -490,7 +481,7 @@ func TestLoadBlockchainfromDB(t *testing.T) {
 	require.Nil(addTestingTsfBlocks(bc))
 	err = bc.Stop(ctx)
 	require.NoError(err)
-	require.Equal(22, ms.Counter())
+	require.Equal(24, ms.Counter())
```

Comment:
Why does it become 24?

Reply:
Because of two votes changed to transfers